decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -573,11 +579,10 @@ def generate_localized_title(field = nil, skip_injection: false)
   factory :coauthorship, class: "Decidim::Coauthorship" do
     transient do
       skip_injection { false }
```

Comment:
Let's leave this as it is in develop please
```suggestion
      skip_injection { false }
    end
    coauthorable { create(:dummy_resource, skip_injection: skip_injection) }
    transient do
```